### PR TITLE
Let scripts use the CLI to internally access the API by default.

### PR DESCRIPTION
**Pull request by eldering**
This simplifies usage, and is typically what you want when you run these commands locally on the domserver.

### CODE REVIEW

**Comment by vmcj** [comment]
There is a typo in the commit message.

**Review by nickygerritsen — APPROVED** [review]

File: doc/manual/import.rst [hunk 1]
```diff
@@ -10,7 +10,7 @@ in your `.netrc`_ file. You need to install `httpie`_ and replace the
 ``<API_URL>`` in the examples below with the API URL of your local DOMjudge
 installation.
 
-For using the CLI, you need to replace ``<WEBAPP_DIR>`` with the path to
+To use the CLI, you need to replace ``<WEBAPP_DIR>`` with the path to
```
Comment (vmcj) [inline doc/manual/import.rst:13]: I might be wrong, but I think we don't mention `<WEBAPP_DIR>` anymore after your commit.

So I propose something like:
```suggestion
To use the CLI, you need to run inside the ``webapp`` directory of the DOMserver.
```

If I understand how the current CLI is implemented.

Comment (nickygerritsen) [reply doc/manual/import.rst:13]: Yea so we drop this whole line imho

Comment (nickygerritsen) [reply doc/manual/import.rst:13]: Ah yes you are right. Sorry, this uses console except for in `Putting all API imports together`

**Pull request merged**